ts/features/authentication/login/cie/screens/CiePinScreen.tsx

```diff
@@ -79,15 +86,23 @@ const onOpenForgotPinPage = () => openWebUrl(pinPukHelpUrl);
 
 const CiePinScreen = () => {
   usePreventScreenCapture();
-  useOnFirstRender(() => {
-    trackLoginCiePinScreen();
-  });
+
+  const { navigateToCieCardReaderScreen, navigateToCieConsentDataUsage } =
+    useActiveSessionLoginNavigation();
 
   const { error } = useIOToast();
   const { name: routeName } = useRoute();
 
   const dispatch = useIODispatch();
 
+  const isActiveSessionLogin = useIOSelector(isActiveSessionLoginSelector);
+
+  useOnFirstRender(() => {
+    if (!isActiveSessionLogin) {
+      trackLoginCiePinScreen();
+    }
```

Comment:
Do you already know the event names and properties? If so, would moving the conditional logic inside the tracking functions help simplify handling both names and properties in one place, while also making the code more readable by avoiding many external if statements? What do you think?